java/bulk-export/bulk-export-task-execution/src/main/java/sleeper/bulkexport/taskexecution/ECSBulkExportTaskRunner.java

```diff
@@ -88,21 +85,21 @@ public static void main(String[] args) throws ObjectFactoryException, IOExceptio
 
         String s3Bucket = args[0];
         Instant startTime = Instant.now();
-        AmazonDynamoDB dynamoDBClient = buildAwsV1Client(AmazonDynamoDBClientBuilder.standard());
-        AmazonSQS sqsClient = buildAwsV1Client(AmazonSQSClientBuilder.standard());
-        AmazonS3 s3Client = buildAwsV1Client(AmazonS3ClientBuilder.standard());
+        DynamoDbClient dynamoDBClient = buildAwsV2Client(DynamoDbClient.builder());
+        SqsClient sqsClient = buildAwsV2Client(SqsClient.builder());
+        S3Client s3Client = buildAwsV2Client(S3Client.builder());
         InstanceProperties instanceProperties = S3InstanceProperties.loadFromBucket(s3Client, s3Bucket);
         TablePropertiesProvider tablePropertiesProvider = S3TableProperties.createProvider(instanceProperties, s3Client,
                 dynamoDBClient);
         try {
```

Comment:
The clients can be in a try-with-resources here. You'll need to move the instance properties and table properties provider setup inside the try/finally.

With individual close statements, if one client fails to close any clients after that will not be closed. With a try-with-resources that's handled automatically. Take a look at how it's done in ECSCompactionTaskRunner.